staging/src/k8s.io/apimachinery/pkg/util/validation/field/errors.go

```diff
@@ -352,3 +370,13 @@ func (list ErrorList) Filter(fns ...utilerrors.Matcher) ErrorList {
 	// FilterOut takes an Aggregate and returns an Aggregate
 	return fromAggregate(err.(utilerrors.Aggregate))
 }
+
+// ExtractCoveredByDeclarative returns a new ErrorList containing only the errors that should be covered by declarative validation.
+func (list ErrorList) ExtractCoveredByDeclarative() ErrorList {
+	return list.Filter(func(err error) bool {
```

Comment:
This Filter method doesn't seem to be used from anywhere, I wonder if we can delete it and just make this function be simpler?